components/gitpod-cli/cmd/tasks-list.go

```diff
@@ -48,11 +49,36 @@ var listTasksCmd = &cobra.Command{
 			2: tablewriter.FgHiBlackColor,
 		}
 
+		mapCurrentToColor := map[int]int{
+			0: tablewriter.FgWhiteColor,
+			1: tablewriter.FgHiGreenColor,
+		}
+
 		for _, task := range tasks {
 			colors := []tablewriter.Colors{}
 
+			ppid := int64(os.Getppid())
+
+			isCurrent := 0
+
+			if task.State == 1 {
+				terminalClient, err := supervisor_helper.GetTerminalServiceClient(context.Background())
+				if err != nil {
+					log.Fatalf("cannot get terminal service: %s", err)
+				}
+
+				terminal, err := terminalClient.Get(context.Background(), &supervisor.GetTerminalRequest{Alias: task.Terminal})
+				if err != nil {
+					panic(err)
+				}
+
+				if ppid == terminal.Pid {
+					isCurrent = 1
```

Comment:
Haven't tested, but I believe you can also use a boolean mapping to make the variable more coherent:

```suggestion
		mapCurrentToColor := map[bool]int{
			false: tablewriter.FgWhiteColor,
			true: tablewriter.FgHiGreenColor,
		}

		for _, task := range tasks {
			colors := []tablewriter.Colors{}

			ppid := int64(os.Getppid())

			isCurrent := false

			if task.State == 1 {
				terminalClient, err := supervisor_helper.GetTerminalServiceClient(context.Background())
				if err != nil {
					log.Fatalf("cannot get terminal service: %s", err)
				}

				terminal, err := terminalClient.Get(context.Background(), &supervisor.GetTerminalRequest{Alias: task.Terminal})
				if err != nil {
					panic(err)
				}

				if ppid == terminal.Pid {
					isCurrent = true
```

Reply:
I've followed your suggestion, @felladrin, I agree, it's nicer :)